### PR TITLE
build: ship React components as `.mjs` without Babel

### PR DESCRIPTION
**Pull request by Robbert**

### CODE REVIEW

File: packages/components-react/textbox-react/package.json [hunk 1]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/textbox-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/textbox-react/package.json:34]: Here as well

File: packages/components-react/button-react/package.json [hunk 2]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/button-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/button-react/package.json:34]: I believe the rollup can also be removed.

File: packages/components-react/calendar-react/package.json [hunk 3]
```diff
@@ -19,30 +19,19 @@
...
     "@types/jest": "29.5.13",
-    "@types/lodash.chunk": "4.2.9",
+    "@types/lodash-es": "4.17.12",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/calendar-react/package.json:34]: Here as well

File: packages/components-react/calendar-react/src/insert-style.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+export const insertStyle = function (css: string) {
+  if (typeof document !== 'undefined') {
+    var head = document.head || document.getElementsByTagName('head')[0];
+    var style = document.createElement('style');
+    style.appendChild(document.createTextNode(css));
+    head.appendChild(style);
+  }
+};
```
Comment (AliKdhim87) [inline packages/components-react/calendar-react/src/insert-style.ts:8]: How about having this function as a utility under something like this @utrecht/utils.

File: packages/components-react/checkbox-group-react/package.json [hunk 5]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/form-field-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/checkbox-group-react/package.json:34]: Here as well

File: packages/components-react/checkbox-react/package.json [hunk 6]
```diff
@@ -19,32 +19,20 @@
...
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/checkbox-css": "workspace:*",
     "@utrecht/custom-checkbox-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/checkbox-react/package.json:35]: Here as well

File: packages/components-react/combobox-react/package.json [hunk 7]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/combobox-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/combobox-react/package.json:34]: Here as well

File: packages/components-react/fieldset-react/package.json [hunk 8]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/form-fieldset-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/fieldset-react/package.json:34]: Here as well

File: packages/components-react/form-field-checkbox-react/package.json [hunk 9]
```diff
@@ -19,30 +19,18 @@
...
     "@testing-library/react": "16.0.1",
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/form-field-checkbox-react/package.json:33]: Here as well

File: packages/components-react/form-field-description-react/package.json [hunk 10]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/form-field-description-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/form-field-description-react/package.json:34]: Here as well

File: packages/components-react/form-label-react/package.json [hunk 11]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/form-label-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/form-label-react/package.json:34]: Here as well

File: packages/components-react/link-react/package.json [hunk 12]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/link-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/link-react/package.json:34]: Here as well

File: packages/components-react/listbox-react/package.json [hunk 13]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/listbox-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/listbox-react/package.json:34]: Here as well

File: packages/components-react/radio-button-react/package.json [hunk 14]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/radio-button-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/radio-button-react/package.json:34]: Here as well

File: packages/components-react/radio-group-react/package.json [hunk 15]
```diff
@@ -19,31 +19,19 @@
...
     "@testing-library/user-event": "14.5.1",
     "@types/jest": "29.5.13",
     "@types/react": "18.3.3",
     "@types/testing-library__jest-dom": "5.14.9",
+    "@utrecht/build-utils-react": "workspace:*",
     "@utrecht/form-field-css": "workspace:*",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "react": "18.3.1",
     "rollup": "4.23.0",
```
Comment (AliKdhim87) [inline packages/components-react/radio-group-react/package.json:34]: Here as well

File: packages/components-react/button-react/package.json (hunk 2 above)
Comment (Robbert) [reply packages/components-react/button-react/package.json:34]: We can't remove it, because `rollup` is still in `pnpm run build`

**Pull request merged**